src/cmd/sample.rs

```diff
@@ -108,8 +111,19 @@ fn sample_reservoir<R: io::Read>(
         reservoir.push(row?);
     }
 
+    // Seeding rng
+    let mut rng: StdRng = match seed {
+        Some(s) => {
+            let s: &[_] = &[s];
+            SeedableRng::from_seed(s)
```

Comment:
Hmm can you just write `SeedableRng::from_seed(&[s])` instead?

Reply:
If I do so I get the following error when compiling:

```
the trait `rand::SeedableRng<&[usize; 1]>` is not implemented for `rand::StdRng`
```

Reply:
Oh, hah. Sigh. Type inference fail. I guess just do `&[s][..]`.

Reply:
Done.